lib/algo/DMD.C

```diff
@@ -427,4 +437,15 @@ DMD::createSnapshotMatrix(std::vector<Vector*> snapshots)
     return snapshot_mat;
 }
 
+void
+DMD::summary(std::string output_path)
+{
+    CSVDatabase* csv_db(new CSVDatabase);
```

Comment:
should we wrap this inside an if statement so that only processor zero outputs this stuff?

Reply:
Right now it is controlled by wrapping the call of this routine with an if statement in the main, following `writeSP` in Laghos. I can quickly put it inside the routine.

Reply:
Sorry, one last thing.


    /**
     * @brief The rank of the process this object belongs to.
     */
    int d_rank;

There is already a d_rank in the class, so we can just use that and pass in only the output_path like you had before.